misk/src/main/kotlin/misk/tracing/backends/noop/NoopTracingModule.kt

```diff
@@ -6,7 +6,7 @@ import io.opentracing.Tracer
 import io.opentracing.noop.NoopTracerFactory
 import misk.inject.KAbstractModule
 
-class NoopTracerBackendModule : KAbstractModule() {
+class NoopTracingModule : KAbstractModule() {
```

Comment:
Is this still needed now that all the tracing stuff is split out?

Reply:
Yeah. Though I need to find a nicer way.